tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -496,7 +496,7 @@ private:
          void Advance()
          {
             auto fnNext_Contig = [&]() {
-               auto &iter = *static_cast<unsigned char **>(fIterator), p = iter;
+               auto &iter = reinterpret_cast<unsigned char *&>(fIterator), p = iter;
```

Comment:
This change I'm not sure I understand. Perhaps add a comment.

Reply:
Done :+1: (note that the comment refers to the updated documentation for `TVirtualCollectionProxy` in PR #13307).